var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -0,0 +1,75 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+import os
+import sys
+from spack import *
+
+
+class Xsdk(Package):
+    """Xsdk is a suite of Department of Energy (DOE) packages for numerical simulation.
+       This is a Spack dummy packages that installs the xSDK packages but 
+       does not itself install anything.
+    """
+
+    homepage = "http://xsdk.info"
+
+    # Dummy url since Spack complains if I don't list something
+    url      = 'https://bitbucket.org/saws/saws/get/master.tar.gz'
+    version('0.1.0','941a541bdf625856be18c9752249146d')
+    # this is for next planned release
+    version('0.2.0','941a541bdf625856be18c9752249146d')
+    # this is for testing with the master of each package 
+    version('master','941a541bdf625856be18c9752249146d')
+
+    # We meed to provide a verions('master' for each package below?
+
+    variant('trilinos', default=True, description='Enables turning off building Trilinos for testing; since it takes so long to build')
+
+    # we actually want 2.11.1 but that isn't available in the Spack file
+    depends_on('hypre@2.10.1~internal-superlu')
```

Comment:
You probably eventually want:

```python
depends_on('hypre@2.10.1~internal-superlu', when='@0.1.0')
```

Then you can add another `depends_on` for Hypre for `0.2.0` when it comes out.

I have ideas about making this kind of thing easier to specify (some kind of version matrix literal) but haven't implemented them yet.